src/assets/stylesheets/theme/box-comment.scss

```diff
@@ -6,7 +6,7 @@
     min-width: 76px;
     img {
       width: 60px;
-      border-radius: 5px;
+      border-radius: var(--jr-card-border-radius);
```

Comment:
* [ ] FYI For variables we use in other places than cards the `--jr-card-` prefix is weird.

Reply:
I'll address this in a follow up with a review of Alex G.